task/build-image-index/0.1/build-image-index.yaml

```diff
@@ -143,7 +153,60 @@ spec:
         cat "image-digest"
       } > "$(results.IMAGE_REF.path)"
       echo -n "${image_manifests:1:-1}" > "$(results.IMAGES.path)"
+
+      IMAGE_DIGEST=$(cat image-digest)
+
+      INDEX_IMAGE_PULLSPEC="${IMAGE}@${IMAGE_DIGEST}"
+      buildah manifest inspect "$INDEX_IMAGE_PULLSPEC" > /index-build-data/manifest_data.json
     securityContext:
       capabilities:
         add:
           - SETFCAP
+
+  - image: quay.io/redhat-appstudio/sbom-utility-scripts-image@sha256:3b219e0610c06401bb5bd355a4bdfeb7f6700f2bef66f89316739d4aae96c89d
+    name: create-sbom
+    computeResources:
+      limits:
+        memory: 512Mi
+        cpu: 200m
+      requests:
+        memory: 256Mi
+        cpu: 100m
+    volumeMounts:
+      - name: shared-dir
+        mountPath: /index-build-data
+    script: |
+      IMAGE_URL="$(cat $(results.IMAGE_URL.path))"
+      IMAGE_DIGEST="$(cat $(results.IMAGE_DIGEST.path))"
+      echo "Creating SBOM result file..."
+      python3 index_image_sbom_script.py \
+        --image-index-url "$IMAGE_URL" \
+        --image-index-digest "$IMAGE_DIGEST" \
+        --inspect-input-file /index-build-data/manifest_data.json \
+        --output-path /index-build-data/sbom-results.json
+    securityContext:
+      runAsUser: 0
```

Comment:
I don't think the `runAsUser: 0` would be needed here, the step just writes a file into a pod-local volume